src/Tribe/Utils/DST.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+
+class Tribe__Events__Utils__DST {
+
+	/**
+	 * Whether the time is in DST or not.
+	 *
+	 * @var bool
+	 */
+	protected $_in_dst;
+
+	/**
+	 * @var string A UNIX timestamp
+	 */
+	protected $time;
+
+	/**
+	 * @var bool
+	 */
+	protected $is_dst_cache;
+
+	/**
+	 * Tribe__Events__Utils__DST constructor.
+	 *
+	 * @param int|string $time    Either a UNIX timestamp or an english format date.
+	 * @param null|bool  $_in_dst An injectable DST status, meant for tests.
+	 */
+	public function __construct( $time, $_in_dst = null ) {
+		if ( ! is_numeric( $time ) ) {
+			$time = strtotime( $time );
+		}
+
+		$this->time    = $time;
+		$this->_in_dst = $_in_dst;
+	}
+
+	/**
+	 * Whether the current time is in DST or not.
+	 *
+	 * @return bool
+	 */
+	public function is_in_dst() {
+		//wwid
```

Comment:
Really minor, but could we expand this comment or remove it if it isn't required? Just because it could cause puzzlement if someone's troubleshooting this method in the future and like me they don't know what it means (...[acronyms.thefreedictionary.com](http://acronyms.thefreedictionary.com/WWID) suggested it might be _"What Would Ivan Do?"_ but I wasn't sure if that applied here :stuck_out_tongue: )